rqt_gps_rtk_plugin/GPS_RTK.config

```diff
@@ -0,0 +1,2 @@
+// Add predefined macros for your project here. For example:
+// #define THE_ANSWER 42
```

Comment:
Awesome :laughing:

Reply:
Yes... but actually this is a Qt Creator specific file and needs to be removed.